lnrpc/rpc.proto

```diff
@@ -1161,6 +1161,13 @@ message Channel {
 
     /// A set of flags showing the current state of the channel.
     string chan_status_flags = 19 [json_name = "chan_status_flags"];
+
+    /// The minimum satoshis this node is required to reserve in its balance.
+    int64 local_chan_reserve_sat = 20 [json_name = "local_chan_reserve_sat"];
+
+    /// The minimum satoshis the other node is required to reserve in its 
+    /// balance.
```

Comment:
Multi-line comments in the protos should use `/**/` instead.

Reply:
Fixed.